.gitignore

```diff
@@ -1 +1,2 @@
 /docs/generated/OJP-prep.xml
+docs/generated/tmp0000.html
```

Comment:
Never had this file, where does it come from?

Reply:
It comes from doing merges locally. I will remove the comment.